src/ui/ManageCookiesDialog/index.svelte

```diff
@@ -89,22 +92,56 @@
       />
     </Section>
   </div>
-  <div class="bottom row">
-    <div class="btn-1 btn--s" on:click={onSaveClick}>Save cookie settings</div>
-    <div class="btn-2 btn--s mrg-m mrg--l" on:click={onAllowAllClick}>Allow all</div>
+  <div class="bottom row txt-center {$IsMobile ? 'body-2' : ''}">
+    <div class="btn--s {$IsMobile ? 'btn-2 mrg-m mrg--t' : 'btn-1'}" on:click={onSaveClick}>
+      Save cookie settings
+    </div>
+    <div class="btn--s {$IsMobile ? 'btn-1' : 'btn-2 mrg--l mrg-m'}" on:click={onAllowAllClick}>
+      Allow all
+    </div>
   </div>
 </Dialog>
 
-<style>
+<style lang="scss">
   .dialog {
     max-width: 600px !important;
+
+    :global(body:not(.desktop)) & {
+      height: 100%;
+
+      h2 {
+        padding: 14px 16px;
+      }
+    }
   }
+
   .cookies {
     padding: 20px 24px 0;
     height: 393px;
-    overflow: auto;
+    overflow-y: auto;
+
+    :global(body:not(.desktop)) & {
+      height: 100%;
+      padding: 24px 16px 0;
+    }
   }
+
+  p {
+    color: var(--fiord);
+  }
+
+  .btn--s {
+    :global(body:not(.desktop)) & {
+      padding: 8px 0;
+    }
+  }
+
   .bottom {
     padding: 20px 24px;
+
+    :global(body:not(.desktop)) & {
```

Comment:
Add single `:global(body:not(.desktop))` at the bottom with necessary selectors in the body